index.html

```diff
@@ -44,6 +44,187 @@ <h3 class="headline-intro">
       
     </div>
     <!--Headline Ends-->
+    <!--Recent Works Section Begins-->
+    <div class="works-container">
+      <div class="works-title">
+        <h2>My Recent Works</h2>
+        <hr>
+      </div>
+      <div class="works-cards">
+        <div class="works-card-1">
+          <img src="./images/Img_Placeholder-card_1.png" alt="card-1-image">
+          <h4>Multi-Post Stories</h4>
+          <div class="card-content-1">A daily selection of privately personalized<br>
+            reads; no accounts or sign-ups<br>
+            has been the industry's standard dummy<br>
+            text ever since the 1500s, when an<br>
+            unknown printer took a standard dummy text.
+          </div>
```

Comment:
- [ ] In line with what I suggested above, using the right semantic tags can help code readability. Please use a `p` tag here instead of a `div`